.github/workflows/main.yml

```
@@ -24,7 +24,7 @@ jobs:
         architecture: 'x64'
     - name: Install dependencies
       run: |
-        python -m pip install jupyterlab==3.0.3
+        python -m pip install jupyterlab==3.0.7
```

Does the webpack change mean we could also use the following?

```suggestion
        python -m pip install jupyterlab~=3.0.7
```

---

yes @jtpio we could do that. let me apply the change. thanks!